Fw/Sm/README

```diff
@@ -0,0 +1,3 @@
+ComPortAi.xml - Port description for communication buffers
```

Comment:
This file looks like it was incorrectly copied from somewhere else.

Reply:
Fixed